recipes-bsp/u-boot/u-boot-toradex_%.bbappend

```diff
@@ -0,0 +1,11 @@
+COMPATIBLE_MACHINE = "(colibri-imx7|colibri-imx7-mender)"
```

Comment:
Should this perhaps go into https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-toradex-nxp/recipes-bsp/u-boot/u-boot-toradex_%25.bbappend instead? It looks pretty applicable everywhere, doesn't it?

Pinging @drewmoseley as well.

Reply:
Well the u-boot-toradex recipe itself is going to be more broadly applicable. And I can see this quickly getting ported to other Toradex boards so I wonder if restricting it here is worthwhile?

Reply:
Ok then.